ompi/mpi/c/finalize.c

```diff
@@ -23,6 +23,9 @@
 #include "ompi/mpi/c/bindings.h"
 #include "ompi/runtime/params.h"
 #include "ompi/errhandler/errhandler.h"
+#include "ompi/runtime/ompi_spc.h"
+
+#include "ompi/runtime/params.h"
```

Comment:
You dont need this, the header is already included 2 lines above.

Reply:
Whoops, ok I've fixed that.